api/text/models.py

```diff
@@ -18,8 +19,28 @@ class Category(models.Model):
                                             blank=False)
 
 
-class Text(models.Model):
-    body = models.TextField(null=False, blank=False)
+class TextComponent(models.Model):
+
+    def init(self) -> None:
+        pass
+
+    def add(self, text_compoment) -> None:
+        pass
+    
+    def get_type(self) -> str:
+        pass
+    
+    def get_price(self) -> float:
+        pass
+
+    def get_fragments(self) -> []:
```

Comment:
Does this abstract method really should be inherited by TextFragment class objects?